waypoint_planner/src/astar_avoid/astar_avoid.cpp

```diff
@@ -210,16 +209,41 @@ void AstarAvoid::run()
 
 bool AstarAvoid::checkInitialized()
 {
-  bool initialized = false;
-
   // check for relay mode
-  initialized = (current_pose_initialized_ && closest_waypoint_initialized_ && base_waypoints_initialized_ &&
-                 (closest_waypoint_index_ >= 0));
```

Comment:
Why did you remove the last condition `closest_waypoint_index_ >= 0`?

Reply:
Good question, I just realized this PR comes after this other one: https://github.com/Autoware-AI/core_planning/pull/36
It's removed in the other PR. I updated the PR description to review/merge #36 first.

Reply:
Understood. Thank you.